go.mod

```diff
@@ -35,6 +36,7 @@ require (
 	github.com/mholt/archiver/v3 v3.5.1
 	github.com/moby/moby v24.0.9+incompatible
 	github.com/opencontainers/image-spec v1.1.0
+	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
```

Comment:
Not super in love with adding this repo that hasn't been maintained, is there anything else out there that we could use?

Reply:
I just switched to hardcoding the ports. It's unlikely we'll see conflicts